Jenkinsfile

```
@@ -12,7 +12,7 @@ pipeline {
     DOCKER_REGISTRY_SECRET = 'secret/observability-team/ci/docker-registry/prod'
     REGISTRY = 'docker.elastic.co'
     STAGING_IMAGE = "${env.REGISTRY}/observability-ci"
-    GO_VERSION = '1.16.4'
+    GO_VERSION = '1.15.14'
```

Bump automation cares only for minor Changes, so if using a patch bump then this is required to be unchanged:

```suggestion
    GO_VERSION = '1.16.4'
```